### PR TITLE
Book dash feb 2020 report draft

### PR DESCRIPTION
**Pull request by malvikasharan**
## Summary

* Adding the approved draft of the book dash report

### List of changes proposed in this PR (pull-request)

* Draft added
* Images will be added to this branch
* Waiting for feedback on remote participation

### What should a reviewer concentrate their feedback on?

- [ ] Content, typos, information
- [ ] Everything looks ok?

### Acknowledging contributors

- [ ] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file.

### CODE REVIEW

File: workshops/book-dash/book-dash-feb20-report.md [hunk 1]
```diff
@@ -0,0 +1,405 @@
...
+One of the feedback from our book dash attendee was that a book dash "closer to home" will be desirable. 
+We are open to explore this idea further, so contact us if you would like to host one of our future book dashes in your institute or organisation. 
+We will be happy to explore interests of researchers in the institutes near you in contributing to _The Turing Way_.
+
+### Represent _The Turing Way_ in your community/conference
+
+If you are attending an Open Research conference, and would like to represent our community, please get in touch with _The Turing Way_ team. 
+They will be happy to help you connect your work with the project, assist you in drafting an abstract for the event and help you prepare to deliver a short workshop or presentation.
+
+There is one upcoming opportunity to attend [NeuroDoWo - the Neurobiology Doctoral Students Workshop](https://neurodowo.nwg-info.de/) taking place in Cologne, Germany from 19 to 21 August 2020.
```
Comment (jezcope) [inline workshops/book-dash/book-dash-feb20-report.md:310]: This opportunity is past now, so maybe remove it?

File: workshops/book-dash/book-dash-feb20-report.md [hunk 2]
```diff
@@ -0,0 +1,405 @@
...
+### Attend online Collaboration Cafes
+
+Our [online Collaboration Cafes](https://github.com/alan-turing-institute/the-turing-way/blob/master/project_management/online-collaboration-cafe.md) take place every 1st Wednesdays at 15:00 GMT and 3rd Wednesdays at 19:00 GMT. 
+These calls are 2 hours long co-working spaces for _The Turing Way_ community. 
+Everyone interested in learning from or developing this project can join us for any duration possible for them. 
+At these calls you can work on your contributions to project. 
+
+You can sign up on this HackMD to attend the [upcoming calls](https://github.com/alan-turing-institute/the-turing-way/blob/master/project_management/online-collaboration-cafe.md): https://hackmd.io/@KirstieJane/CollabCafe. 
+If you would like to host these calls at other times, please get in touch with the team members. 
+The next calls are on [4 March 2020 at 15:00 GMT](https://arewemeetingyet.com/London/2019-03-04/15:00/TuringWay-CollaborationCafe) and [19 March 2020 at 19:00 GMT](https://arewemeetingyet.com/London/2019-03-19/19:00/TuringWay-CollaborationCafe).
```
Comment (jezcope) [inline workshops/book-dash/book-dash-feb20-report.md:285]: Maybe remove this line as it's now out of date.

**Review by jezcope — COMMENTED** [review]
I've submitted some grammar, punctuation & spelling changes and made a couple of suggestions. Otherwise I think it looks great, very clear description!

**Comment by malvikasharan** [comment]
Thanks, Jez. Great points on probably avoiding such "time-sensitive" information that are likely to get outdated after the reports are published. made the changes. 🙌

**Review by jezcope — APPROVED** [review]
I think this is good to go! :rocket:

**Comment by malvikasharan** [comment]
Yay!!! Thanks a lot.

**Pull request merged**